permute/tests/test_npc.py

```diff
@@ -137,22 +137,34 @@ def med_diff(data, resp_index):
     test_array = Experiment.make_test_array(med_diff, [0, 1])
     res = sim_npc(data, test_array, combine="fisher", seed=None, reps=int(1000))
     np.testing.assert_almost_equal(res[0], 1)
-    
+    res = sim_npc(data, test_array, combine="liptak", seed=1234, reps=int(1000))
+    np.testing.assert_almost_equal(res[0], 1)
+    res = sim_npc(data, test_array, combine=tippett, in_place=True, seed=None, reps=int(1000))
+    np.testing.assert_almost_equal(res[0], 1)
+
```

Comment:
In above code, I added prints for fisher, liptak, tippet combine methods. 
So res in line 138 is printed as "Res1:".  
res in line 140 is printed as "Res2:".
res in line 142 is printed as "Res3:".

```
Res1: (1.0, {0: -1.0, 1: -1.0}, {0: 1.0, 1: 1.0})
Res2: (1.0, {0: -1.0, 1: -1.0}, {0: 1.0, 1: 1.0})
Res3: (1.0, {0: -1.0, 1: -1.0}, {0: 1.0, 1: 1.0})
.

```

Have added the code that I used to print:

```
    print("")
    print("Res1: {}".format(res))
    np.testing.assert_almost_equal(res[0], 1)
    res = sim_npc(data, test_array, combine="liptak", seed=1234, reps=int(1000))
    print("Res2: {}".format(res))
    np.testing.assert_almost_equal(res[0], 1)
    res = sim_npc(data, test_array, combine=tippett, in_place=True, seed=None, reps=int(1000))
    print("Res3: {}".format(res))
    np.testing.assert_almost_equal(res[0], 1)

```